tests/integration/components/donation/credit-card-test.js

```diff
@@ -1,29 +1,64 @@
 import { moduleForComponent, test } from 'ember-qunit';
+import Ember from 'ember';
 import hbs from 'htmlbars-inline-precompile';
+import PageObject from 'ember-cli-page-object';
+import stubService from 'code-corps-ember/tests/helpers/stub-service';
+
+import creditCardComponent from '../../../pages/components/donation/credit-card';
+
+let page = PageObject.create(creditCardComponent);
+
+const {
+  K
+} = Ember;
+
+let setHandler = function(context, submitHandler = K) {
+  context.set('submitHandler', submitHandler);
+};
 
 moduleForComponent('donation/credit-card', 'Integration | Component | donation/credit card', {
-  integration: true
+  integration: true,
+  beforeEach() {
+    setHandler(this);
+    page.setContext(this);
+  },
+  afterEach() {
+    page.removeContext();
+  }
 });
 
-test('inputs can be disabled', function(assert) {
-  assert.expect(3);
-  this.set('canDonate', false);
-  this.render(hbs`{{donation/credit-card canDonate=canDonate}}`);
+test('it sends submit with credit card fields when button is clicked', function(assert) {
+  assert.expect(2);
 
-  let inputs = this.$().find('input').get();
+  this.set('canDonate', true);
 
-  inputs.forEach((input) => {
-    assert.ok($(input).prop('disabled'));
-  });
-});
+  let expectedProps = {
+    cardNumber: '1234-5678-9012-3456',
+    cvc: '123',
+    month: '12',
+    year: '2020'
+  };
 
-test('inputs are enabled by default', function(assert) {
-  assert.expect(3);
-  this.render(hbs`{{donation/credit-card}}`);
+  let submitHandler = function(actualProps) {
+    assert.deepEqual(actualProps, expectedProps, 'Action was called with proper attributes.');
+  };
 
-  let inputs = this.$().find('input').get();
+  setHandler(this, submitHandler);
 
-  inputs.forEach((input) => {
-    assert.notOk($(input).prop('disabled'));
+  stubService(this, 'stripe', {
+    card: {
+      validateCardNumber: () => true,
```

Comment:
Resolved in https://github.com/code-corps/code-corps-ember/pull/668/commits/37a6024037b25a58e0cd6c4d0abc92c1667f3cce as `Ember.K` is no longer needed.